packages/roosterjs-editor-plugins/lib/CustomReplace/CustomReplace.ts

```diff
@@ -0,0 +1,157 @@
+import { Editor, EditorPlugin } from 'roosterjs-editor-core';
+import { PluginEvent, PluginEventType, PluginKeyPressEvent } from 'roosterjs-editor-types';
+
+type Replacement = {
+    sourceString: string,
+    replacementHTML: string,
+    matchSourceCaseSensitive: boolean
+}
+
+const defaultReplacements: Replacement[] = [
+    {
+        sourceString: ':)',
+        replacementHTML: '🙂',
+        matchSourceCaseSensitive: true,
+    },
```

Comment:
suggest create a local util function to help create this object in order to save some bytes. e.g.
function createReplacements(sourceString: string, ... ) {
  return { ... }
}

Reply:
We have uglified build, and you can find the build result from dist\roosterjs\dist\rooster-min.js.
However, this won't minify the interface members. If you search in the minified file you will still see those member names there.